google/cloud/security/common/data_access/violation_format.py

```diff
@@ -31,118 +28,13 @@ def format_policy_violation(violation):
-
-    Yields:
-        A tuple of the rule violation properties.
-    """
-
-    resource_type = violation.resource_type
-    if resource_type:
-        resource_type = resource_type[:255]
-
-    resource_id = violation.resource_id
-    if resource_id:
-        resource_id = str(resource_id)[:255]
-
-    rule_name = violation.rule_name
-    if rule_name:
-        rule_name = rule_name[:255]
-
-    role = violation.role
-    if role:
-        role = role[:255]
-
-    iam_members = violation.members
-    if iam_members:
-        members = [str(iam_member)[:255] for iam_member in iam_members]
-    else:
-        members = []
-
-    for member in members:
-        yield (resource_type,
-               resource_id,
-               rule_name,
-               violation.rule_index,
-               violation.violation_type,
-               role,
-               member)
-
-def format_buckets_acl_violation(violation):
-    """Format the bucket acls violation data into a tuple.
-
-    Also flattens the RuleViolation, since it consists of the resource,
-    rule, and members that don't meet the rule criteria.
-
-    Various properties of RuleViolation may also have values that exceed the
-    declared column length, so truncate as necessary to prevent MySQL errors.
-
-    Args:
-        violation: The bucket acls RuleViolation.
-
-    Yields:
-        A tuple of the rule violation properties.
-    """
-
-    resource_type = violation.resource_type
-    if resource_type:
-        resource_type = resource_type[:255]
-
-    resource_id = violation.resource_id
-    if resource_id:
-        resource_id = str(resource_id)[:255]
-
-    rule_name = violation.rule_name
-    if rule_name:
-        rule_name = rule_name[:255]
-
-    role = violation.role
-    if role:
-        role = role[:255]
-
-    entity = violation.entity
-    if entity:
-        entity = entity[:255]
-
-    email = violation.email
-    if email:
-        email = email[:255]
-
-    domain = violation.domain
-    if domain:
-        domain = domain[:255]
-
-    bucket = violation.bucket
-    if bucket:
-        bucket = bucket[:255]
-
-    yield (resource_type,
-           resource_id,
-           rule_name,
-           violation.rule_index,
-           violation.violation_type,
-           role,
-           entity,
-           email,
-           domain,
-           bucket)
-
-def format_cloudsql_acl_violation(violation):
-    """Format the CloudSQL acls violation data into a tuple.
-
-    Also flattens the RuleViolation, since it consists of the resource,
-    rule, and members that don't meet the rule criteria.
-
-    Various properties of RuleViolation may also have values that exceed the
-    declared column length, so truncate as necessary to prevent MySQL errors.
-
-    Args:
-        violation: The cloudsql acls RuleViolation.
+        violation (namedtuple): The Policy RuleViolation. This is a named
+            tumple with the following attributes 'resource_type','resource_id',
```

Comment:
Feel free to reference the list defined in violation_dao.py here. There's syntax for saying something like "See..."